libs/viewer/src/ViewerGui.cpp

```diff
@@ -523,12 +523,18 @@ void ViewerGui::applyAnimation(double currentTime, FilamentInstance* instance) {
         mResetAnimation = false;
     }
     const double elapsedSeconds = currentTime - mCurrentStartTime;
-    if (numAnimations > 0 && mCurrentAnimation > 0) {
-        animator.applyAnimation(mCurrentAnimation - 1, elapsedSeconds);
-        if (elapsedSeconds < mCrossFadeDuration && mPreviousAnimation > 0) {
+    if (numAnimations > 0 && mCurrentAnimation >= 0) {
```

Comment:
```suggestion
    if (animationCount > 0 && mCurrentAnimation >= 0) {
```

I know the existing code is use `numAnimation` but that was wrong. Please update the name as part of the PR since you're modifying this code. Thank you!